.gitignore

```diff
@@ -129,3 +129,13 @@ override.tf.json
 *_override.tf.json
 .terraformrc
 .terraform.rc
+
+# Local Netlify folder
+.netlify
```

Comment:
When will we have this folder ?  is it necessary for us to add it ?

Reply:
not sure but its already added in master branch .gitignore. so this just aligns with it.